lang/cs/Org.Apache.REEF.IMRU/OnREEF/StateMachine/StateTransition.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+namespace Org.Apache.REEF.IMRU.OnREEF.StateMachine
```

Comment:
In terms of the name space, I can put #938 to the same name space as this one. What do you think? 
For TaskStateTransition , yes, which ever #938 or this one merged first, I will change the code to use the template. 

May be merge this one first then I will change #938